lib/streamaggr/streamaggr.go

```diff
@@ -481,6 +481,9 @@ func newAggregator(cfg *Config, path string, pushFunc PushFunc, ms *metrics.Set,
 		if stalenessInterval < interval {
 			return nil, fmt.Errorf("staleness_interval=%s cannot be smaller than interval=%s", cfg.StalenessInterval, cfg.Interval)
 		}
+		if stalenessInterval%interval != 0 {
```

Comment:
I'm not sure this is really needed. @hagen1778 what do you think?

Reply:
this constraint is documented, I can remove it if you don't like it

Reply:
could you please point me to docs where it's mentioned?

Reply:
it's in streamaggr_test.go:L112 , alternative - I can remove this test

Reply:
removed